### DISCUSSION TITLE
Is there anyone who published the app on Amazon Playstore.

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#805] [General]
Hi,

Friend, I want to upload my app on amazon playstore. But while debugging my app on amazon playstore, my app close as soon it is launched.

I created my app on animate cc 2021 with Harman SDK 33.1.1.444. I shared my app to amazon firestick via app2fire app. App successful installed on amazon firestick but closing as soon launch.

Please Help if any one have successful played app on amazon firestick.

**Comment by ajwfrost** [comment]
Hi
To try to debug the close-on-launch, would it be possible to use ADB to get the logcat output? or, do you have a way to enable developer options and then send us a crash report (or have you got a third party mechanism for this)?
https://www.aftvnews.com/how-to-connect-to-a-fire-tv-or-fire-tv-stick-via-adb/
https://forums.developer.amazon.com/articles/113419/deprecation-of-crash-reporting-in-developer-portal.html

Just to check, which Fire Stick is it? or more importantly, what version of Android is it running? .444 has an issue with older Android builds that could affect the original or second generation FireTV Stick devices..

thanks

**Comment by Michoko92** [comment]
At least it is possible in theory, as I have about 30+ apps on the Amazon store. On a Kindle Fire tablet, they work fine (didn't test on a firestick though, but they should work, as they passed the Amazon review process)

**Comment by balaedit** [comment]
My Android Fire stick had not working properly. I ordered new android fire stick but due to state lockdown, only essential deliveries are available. So I am getting it from other state and so will personally transport it. 

I will back as soon I get my new Amazon firestick.

**Reply by balaedit** [reply]
Meanwhile do anyone have sample (Just some animated) flash file with all the manifest and other description and assets for publishing Amazon Play Store. If anyone who had created just simple any project for amazon playstore, please share it with me at balaedit@gmail.com
